app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ProfilesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :load_user
+
+  def index
+    @user
+  end
+
+  def update
+    if @user.update(profile_params)
```

Comment:
we already had issue with `update` method. It always returns object, which is a true value (false values in ruby: `false`, `nil`). I highly recommend to use `update_attributes`